default/stringtables/en.txt

```diff
@@ -124,6 +124,9 @@ Back
 LAST
 Last
 
+DIFFICULTY
+Difficulty
+
```

Comment:
I didn't want to limit the label to beginner bonus, as higher difficulties might give players penalties instead of bonuses.